src/lib/holocene/combobox/combobox.svelte

```diff
@@ -301,7 +301,9 @@
     </Button>
     {#if $$slots.action}
       <div class="ml-1 h-full w-1 border-l-2 border-subtle" />
```

Comment:
Wait, why is this div closed? Is it supposed to wrap the action slot?

Reply:
Oh, maybe it's just there for the border 🤔 We can probably just add the padding to this then

Reply:
That `div` was the existing border here ⬇️, but realizing I should just refactor this to one `div` 😅 Updated [here](https://github.com/temporalio/ui/pull/2214/commits/816924bd2c8a5b4efc23b2b325b40c80dc59ecd5).

<img width="60" alt="Screenshot 2024-07-17 at 2 14 58 PM" src="https://github.com/user-attachments/assets/566ec80c-9874-44df-9b3c-4035b88943df">